components/notebook-dockerfiles/pytorch/requirements.txt

```diff
@@ -2,31 +2,32 @@
 
 # Jupyter related packages
 jupyter-console==6.4.4
-jupyterlab==3.4.3
-jupyterlab-git==0.37.1
+jupyterlab==3.4.4
+jupyterlab-git==0.38.0
 notebook==6.4.12
-ipykernel==6.15.0
+ipykernel==6.15.1
 
 # Extensions for jupyter notebook
 widgetsnbextension==3.6.1
 ipywidgets==7.7.1
 
 # Kubeflow Related Packages
-kfp==1.8.12
-kfp-server-api==1.8.2
+kfp==1.8.13
+kfp-server-api==1.8.3
 kfserving==0.6.1
 kubeflow-training==1.4.0
+kubeflow-katib==0.14.0
 
 # Common data science packages
 h5py==3.7.0
-pandas==1.4.2
+pandas==1.4.3
 matplotlib==3.5.2
 xgboost==1.6.1
 ipympl==0.9.1
 
 # AWS related packages
-awscli==1.25.15
-boto3==1.24.15
+awscli==1.25.44
+boto3==1.24.44
 
-# pytorch packages
+# pytorch packages, latest version reinstalls Pytorch.
```

Comment:
did some behavior change ? looks like version number is the same

Reply:
oh no, that was more of a note to self, to not update that dependency to the latest version since it updated the pytorch binary as well.